client/src/components/files/FileCard.vue

```diff
@@ -26,17 +26,23 @@
       <ion-icon :icon="ellipsisHorizontal" />
     </div>
     <div class="card-content">
-      <ion-avatar class="card-content__icons">
+      <div class="card-content-icons">
         <ion-icon
-          class="icon-item"
-          :icon="file.isFile() ? document : folder"
+          class="folder-icon"
```

Comment:
Why use `ion-icon` in this case and `ms-image` otherwise? Couldn't it be just one `<ms-image>` tags with props changed depending on the file type?